### PR TITLE
Test/atlas contains element

### PR DESCRIPTION
**Pull request by KemalYildizz**

### CODE REVIEW

File: Testing/Gatling/src/test/resources/io/gatling/tests/atlascontainselement/0006_request.dat [hunk 1]
```diff
@@ -0,0 +1 @@
+["O43z0dpjhgX20SCx4KAo"]
```
Comment (LiWeiYeh) [inline Testing/Gatling/src/test/resources/io/gatling/tests/atlascontainselement/0006_request.dat:1]: Remove atlascontainselement folder

File: Testing/Gatling/src/test/scala/io/gatling/tests/RecordedSimulation.scala [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+
+import scala.concurrent.duration._
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import io.gatling.jdbc.Predef._
+
+class RecordedSimulation extends Simulation {
+
+	val httpProtocol = http
```
Comment (LiWeiYeh) [inline Testing/Gatling/src/test/scala/io/gatling/tests/RecordedSimulation.scala:11]: remove headers and httpProtocol

File: Testing/Gatling/src/test/scala/io/gatling/tests/RecordedSimulation.scala [hunk 3]
```diff
@@ -0,0 +1,255 @@
+package io.gatling.tests
```
Comment (LiWeiYeh) [inline Testing/Gatling/src/test/scala/io/gatling/tests/RecordedSimulation.scala:1]: remove `RecordedSimulation.scala`

File: Testing/Gatling/src/test/scala/io/gatling/tests/requests/RAPRequests.scala [hunk 4]
```diff
@@ -114,15 +114,19 @@ object RAPRequests {
     .delete("/api/v1/resource/SESSION/1/MyScripts/${accountId}/_EMPTY_/${scriptId}")
     .check(status.is(200))
 
-  /*  val getAtlas = http("Go to the atlas page")
+  val getAtlas = http("Go to the atlas page")
       .get("/api/v1/resource/SESSION/1/Atlas")
-      .check(jsonPath("$._id_").saveAs("contextId"))
-      .check(jsonPath("$.Terug_32_naar_32_script._id_").is("scriptId"))
+      //.check(jsonPath("$[0]._id_").saveAs("contextId"))
```
Comment (LiWeiYeh) [inline Testing/Gatling/src/test/scala/io/gatling/tests/requests/RAPRequests.scala:119]: remove comments

File: Testing/Gatling/src/test/scala/io/gatling/tests/requests/RAPRequests.scala [hunk 5]
```diff
@@ -114,15 +114,19 @@ object RAPRequests {
...
+      .check(jsonPath("$[0].Terug_32_naar_32_script[0]._id_").is("${scriptId}"))
       .check(status.is(200))
 
-    val getContext = http("Go to the context page")
+  val checkAtlas = http("Validate Atlas section")
+    .get("/api/v1/resource/SESSION/1/Atlas")
+    .check(jsonPath("$[0].Terug_32_naar_32_script[0]._id_").exists)
+
+  /*val getContext = http("Go to the context page")
```
Comment (LiWeiYeh) [inline Testing/Gatling/src/test/scala/io/gatling/tests/requests/RAPRequests.scala:127]: remove comments

File: Testing/Gatling/src/test/scala/io/gatling/tests/scenarios/RAPScenarios.scala [hunk 6]
```diff
@@ -86,14 +86,17 @@ object RAPScenarios {
...
-     .exec(RAPRequests.getHome)
-     .exec(RAPRequests.getLogin)
-     .exec(RAPRequests.patchCorrectLogin)
-     .exec(RAPRequests.getMyScriptReturns200)
-     /*.exec(RAPRequests.postNewScript)
-     .exec(RAPRequests.patchCorrectCompileScriptContentAndName)
-     .exec(RAPRequests.patchCorrectButtons)*/
-     .exec(RAPRequests.getAtlas)
-     .exec(RAPRequests.getContext)*/
+
```
Comment (LiWeiYeh) [inline Testing/Gatling/src/test/scala/io/gatling/tests/scenarios/RAPScenarios.scala:89]: remove whitespace here

File: Testing/Gatling/src/test/scala/io/gatling/tests/scenarios/RAPScenarios.scala [hunk 7]
```diff
@@ -86,14 +86,17 @@ object RAPScenarios {
...
+    .exec(RAPRequests.getHome)
+    .exec(RAPRequests.getLogin)
+    .exec(RAPRequests.patchCorrectLogin)
+    .exec(RAPRequests.getMyScriptReturns200)
+    .exec(RAPRequests.postNewScript)
+    .exec(RAPRequests.patchCorrectCompileScriptContentAndName)
+    .exec(RAPRequests.getAtlas)
+    .exec(RAPRequests.checkAtlas)
+    .exec(RAPRequests.deleteScript)
+
```
Comment (LiWeiYeh) [inline Testing/Gatling/src/test/scala/io/gatling/tests/scenarios/RAPScenarios.scala:100]: remove whitespace

File: .idea/compiler.xml [hunk 8]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (LiWeiYeh) [inline .idea/compiler.xml:1]: Remove commits from .idea

Comment (KemalYildizz) [reply .idea/compiler.xml:1]: doe jij dit

**Pull request merged**